kuma/wiki/utils.py

```diff
@@ -59,3 +67,42 @@ def tidy_content(content):
         return tidied.decode('utf-8'), errors
     else:
         return content
+
+
+def analytics_user_counts(*revs):
+    """Given some document revision numbers, returns a dict matching those
+    with the number of users Google Analytics thinks has visited each revision.
+
+    """
+
+    scopes = ['https://www.googleapis.com/auth/analytics.readonly']
+    credentials = ServiceAccountCredentials.from_json_keyfile_dict(
+        json.loads(config.GOOGLE_ANALYTICS_CREDENTIALS), scopes=scopes)
+    http_auth = credentials.authorize(Http())
+    service = build('analyticsreporting', 'v4', http=http_auth)
+
+    request = service.reports().batchGet(
+        body={
+            'reportRequests': [
+                # `dimension12` is the custom variable containing a page's rev #.
+                {
+                    'dimensions': [{'name': 'ga:dimension12'}],
+                    'metrics': [{'expression': 'ga:users'}],
```

Comment:
I'm not sure [ga:users](https://developers.google.com/analytics/devguides/reporting/core/dimsmets#view=detail&group=user&jump=ga_users) is the dimension we want. I'm not sure how it is calculated, but will see what the docs say.  [ga:uniquePageviews](https://developers.google.com/analytics/devguides/reporting/core/dimsmets#view=detail&group=page_tracking&jump=ga_uniquepageviews) is the dimension they show in the site content views in GA.